src/stores/pacbioLibraries.js

```diff
@@ -0,0 +1,122 @@
+import { defineStore } from 'pinia'
+import useRootStore from '@/stores'
+import { handleResponse } from '@/api/ResponseHelper.js'
+import { groupIncludedByResource, dataToObjectById } from '@/api/JsonApi.js'
+
+export const usePacbioLibrariesStore = defineStore('pacbioLibraries', {
+  state: () => ({
+    tagSetChoices: [],
+    tagChoices: {},
+    libraries: [],
+  }),
+
+  getters: {
+    // Returns a list of tag options by tagSetId
+    tagChoicesForId: (state) => (tagSetId) => {
+      return state.tagChoices[tagSetId] || []
+    },
+  },
+  actions: {
+    /**Create a new libary */
+    async createLibraryInTraction(library, tagId) {
+      const rootState = useRootStore()
+
+      const body = {
+        data: {
+          type: 'libraries',
+          attributes: {
+            pacbio_request_id: library.sample.id,
+            template_prep_kit_box_barcode: library.template_prep_kit_box_barcode,
+            tag_id: tagId,
+            concentration: library.concentration,
+            volume: library.volume,
+            insert_size: library.insert_size,
+            primary_aliquot_attributes: {
+              template_prep_kit_box_barcode: library.template_prep_kit_box_barcode,
+              volume: library.volume,
+              concentration: library.concentration,
+              insert_size: library.insert_size,
+            },
+          },
+        },
+      }
+      //TODO_LIBRARY_CHANGE:Needs to be changed to library
+      const request = rootState.api.traction.pacbio.libraries
+      const promise = request.create({ data: body, include: 'tube,primary_aliquot' })
+      const { success, data: { included = [] } = {}, errors } = await handleResponse(promise)
+      const { tubes: [tube = {}] = [] } = groupIncludedByResource(included)
+      const { attributes: { barcode = '' } = {} } = tube
+      return { success, barcode, errors }
+    },
+
+    /**Delete all libraries with given library ids */
+    async deleteLibraries(libraryIds) {
```

Comment:
Bit of a side note but I think I am against the ability to delete libraries after we make these changes. Since they will be able to be used in runs directly now it makes library data more valuable. Thoughts? @stevieing @seenanair